### ISSUE TITLE
Crash during playback of AHX file (reproducible)

### ISSUE THREAD

**Issue by bryc** [pete-gordon/hivelytracker#36]
So, on the Windows version of HVL 1.8, load up this file: 

http://phd-sid.ethz.ch/debian/ahx/AHX/Curt%20Cool/rubber%20spine.ahx

It consistently crashes at this exact moment:

![image](https://user-images.githubusercontent.com/1408749/56446156-000b3e80-62cf-11e9-9fcd-ecf1b63bb6b2.png)

Basically it will crash whenever attempting to play Instrument 4. I can't pinpoint the exact issue, but this is what the instrument says:

![image](https://user-images.githubusercontent.com/1408749/56446291-e9191c00-62cf-11e9-96fd-58c2521c6684.png)

**Comment by alexmyczko** [comment]
works fine with linux for me though, i'm even able to import it to klystrack for replay, also no problem...

**Comment by bryc** [comment]
@alexmyczko Then the Win32 version is doing something wrong I suppose.

Think I found the reason. I checked AHX in WinUAE and it worked fine. But it was showing "000" in Decay and Sustain of Instrument 4:

![image](https://user-images.githubusercontent.com/1408749/59706573-b6f04180-91ce-11e9-9bae-51879f1e0b9a.png)

Even in AHX, the lowest possible value is "001", so "000" is beyond that range. HivelyTracker doesn't even show that it is set to 000 in my version.

Simply manually adjusting those back to 001 in HivelyTracker prevents the crash.
